nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/resources/OSGI-INF/commandline-imagemagick-contrib.xml

```diff
@@ -20,9 +20,10 @@
       <installationDirective>You need to install ImageMagick.</installationDirective>
     </command>
 
+
     <command name="resizer" enabled="true">
       <commandLine>convert</commandLine>
-      <parameterString>-define registry:temporary-path=#{nuxeo.tmp.dir} -quiet -depth #{targetDepth} #{inputFilePath}[0] jpg:- | convert - -resize #{targetWidth}x#{targetHeight} #{outputFilePath}</parameterString>
+      <parameterString>-define registry:temporary-path=#{nuxeo.tmp.dir} -quiet -depth #{targetDepth} #{inputFilePath}[0] -resize #{targetWidth}x#{targetHeight} #{outputFilePath}</parameterString>
```

Comment:
This is losing the `jpg:-` part, which forces JPG format. Isn't this an issue?

Reply:
I actually think we should review these improvements more globally if interested. As it looks like its better to convert all the thumbnails to `png` ( instead of `jpeg`) generally

Reply:
We have the same perf issues on our side in AI - +1 for doing a review (we may test on our side)